[2023-12-03 21:24:25,307] {processor.py:153} INFO - Started process (PID=5260) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:24:25,307] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:24:25,307] {logging_mixin.py:115} INFO - [2023-12-03 21:24:25,307] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:24:25,318] {logging_mixin.py:115} INFO - [2023-12-03 21:24:25,318] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:24:25,318] {logging_mixin.py:115} INFO - [2023-12-03 21:24:25,318] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:24:25,318] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:24:25,326] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03 21:24:57,233] {processor.py:153} INFO - Started process (PID=5298) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:24:57,233] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:24:57,233] {logging_mixin.py:115} INFO - [2023-12-03 21:24:57,233] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:24:57,244] {logging_mixin.py:115} INFO - [2023-12-03 21:24:57,243] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:24:57,244] {logging_mixin.py:115} INFO - [2023-12-03 21:24:57,244] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:24:57,244] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:24:57,252] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03 21:25:30,910] {processor.py:153} INFO - Started process (PID=5344) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:25:30,911] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:25:30,911] {logging_mixin.py:115} INFO - [2023-12-03 21:25:30,911] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:25:30,921] {logging_mixin.py:115} INFO - [2023-12-03 21:25:30,921] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:25:30,921] {logging_mixin.py:115} INFO - [2023-12-03 21:25:30,921] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:25:30,921] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:25:30,929] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03 21:26:01,708] {processor.py:153} INFO - Started process (PID=5378) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:26:01,708] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:26:01,709] {logging_mixin.py:115} INFO - [2023-12-03 21:26:01,709] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:26:01,719] {logging_mixin.py:115} INFO - [2023-12-03 21:26:01,719] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:26:01,719] {logging_mixin.py:115} INFO - [2023-12-03 21:26:01,719] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:26:01,719] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:26:01,727] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03 21:26:35,465] {processor.py:153} INFO - Started process (PID=5428) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:26:35,466] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:26:35,466] {logging_mixin.py:115} INFO - [2023-12-03 21:26:35,466] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:26:35,476] {logging_mixin.py:115} INFO - [2023-12-03 21:26:35,476] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:26:35,476] {logging_mixin.py:115} INFO - [2023-12-03 21:26:35,476] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:26:35,476] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:26:35,484] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03 21:27:06,987] {processor.py:153} INFO - Started process (PID=5462) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:27:06,987] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:27:06,987] {logging_mixin.py:115} INFO - [2023-12-03 21:27:06,987] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:27:06,998] {logging_mixin.py:115} INFO - [2023-12-03 21:27:06,998] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:27:06,999] {logging_mixin.py:115} INFO - [2023-12-03 21:27:06,999] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:27:06,999] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:27:07,007] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03 21:27:39,557] {processor.py:153} INFO - Started process (PID=5514) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:27:39,557] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:27:39,558] {logging_mixin.py:115} INFO - [2023-12-03 21:27:39,557] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:27:39,568] {logging_mixin.py:115} INFO - [2023-12-03 21:27:39,568] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:27:39,568] {logging_mixin.py:115} INFO - [2023-12-03 21:27:39,568] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:27:39,568] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:27:39,576] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03 21:28:14,064] {processor.py:153} INFO - Started process (PID=5566) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:28:14,065] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:28:14,065] {logging_mixin.py:115} INFO - [2023-12-03 21:28:14,065] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:28:14,076] {logging_mixin.py:115} INFO - [2023-12-03 21:28:14,076] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:28:14,076] {logging_mixin.py:115} INFO - [2023-12-03 21:28:14,076] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:28:14,076] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:28:14,084] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03 21:28:45,843] {processor.py:153} INFO - Started process (PID=5601) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:28:45,844] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:28:45,844] {logging_mixin.py:115} INFO - [2023-12-03 21:28:45,844] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:28:45,854] {logging_mixin.py:115} INFO - [2023-12-03 21:28:45,854] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:28:45,854] {logging_mixin.py:115} INFO - [2023-12-03 21:28:45,854] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:28:45,854] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:28:45,862] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.021 seconds
[2023-12-03 21:29:18,482] {processor.py:153} INFO - Started process (PID=5642) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:29:18,482] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:29:18,482] {logging_mixin.py:115} INFO - [2023-12-03 21:29:18,482] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:29:18,496] {logging_mixin.py:115} INFO - [2023-12-03 21:29:18,495] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:29:18,496] {logging_mixin.py:115} INFO - [2023-12-03 21:29:18,496] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:29:18,496] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:29:18,505] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2023-12-03 21:29:51,111] {processor.py:153} INFO - Started process (PID=5680) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:29:51,111] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:29:51,112] {logging_mixin.py:115} INFO - [2023-12-03 21:29:51,112] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:29:51,122] {logging_mixin.py:115} INFO - [2023-12-03 21:29:51,122] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:29:51,122] {logging_mixin.py:115} INFO - [2023-12-03 21:29:51,122] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:29:51,122] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:29:51,130] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03 21:30:22,306] {processor.py:153} INFO - Started process (PID=5719) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:30:22,306] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:30:22,307] {logging_mixin.py:115} INFO - [2023-12-03 21:30:22,307] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:30:22,317] {logging_mixin.py:115} INFO - [2023-12-03 21:30:22,317] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:30:22,317] {logging_mixin.py:115} INFO - [2023-12-03 21:30:22,317] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:30:22,317] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:30:22,325] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2023-12-03 21:30:55,947] {processor.py:153} INFO - Started process (PID=5762) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:30:55,948] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:30:55,948] {logging_mixin.py:115} INFO - [2023-12-03 21:30:55,948] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:30:55,960] {logging_mixin.py:115} INFO - [2023-12-03 21:30:55,960] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:30:55,960] {logging_mixin.py:115} INFO - [2023-12-03 21:30:55,960] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:30:55,960] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:30:55,969] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-03 21:31:28,860] {processor.py:153} INFO - Started process (PID=5796) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:31:28,861] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:31:28,861] {logging_mixin.py:115} INFO - [2023-12-03 21:31:28,861] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:31:28,873] {logging_mixin.py:115} INFO - [2023-12-03 21:31:28,872] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:31:28,873] {logging_mixin.py:115} INFO - [2023-12-03 21:31:28,873] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:31:28,873] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:31:28,881] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03 21:31:59,791] {processor.py:153} INFO - Started process (PID=5830) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:31:59,791] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:31:59,791] {logging_mixin.py:115} INFO - [2023-12-03 21:31:59,791] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:31:59,803] {logging_mixin.py:115} INFO - [2023-12-03 21:31:59,803] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:31:59,803] {logging_mixin.py:115} INFO - [2023-12-03 21:31:59,803] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:31:59,803] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:31:59,812] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-03 21:32:33,974] {processor.py:153} INFO - Started process (PID=5865) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:32:33,974] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:32:33,975] {logging_mixin.py:115} INFO - [2023-12-03 21:32:33,974] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:32:33,987] {logging_mixin.py:115} INFO - [2023-12-03 21:32:33,987] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:32:33,987] {logging_mixin.py:115} INFO - [2023-12-03 21:32:33,987] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:32:33,987] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:32:34,003] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.032 seconds
[2023-12-03 21:33:07,654] {processor.py:153} INFO - Started process (PID=5902) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:33:07,654] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:33:07,654] {logging_mixin.py:115} INFO - [2023-12-03 21:33:07,654] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:33:07,665] {logging_mixin.py:115} INFO - [2023-12-03 21:33:07,665] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:33:07,666] {logging_mixin.py:115} INFO - [2023-12-03 21:33:07,665] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:33:07,666] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:33:07,674] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03 21:33:39,434] {processor.py:153} INFO - Started process (PID=5936) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:33:39,435] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:33:39,435] {logging_mixin.py:115} INFO - [2023-12-03 21:33:39,435] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:33:39,447] {logging_mixin.py:115} INFO - [2023-12-03 21:33:39,446] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:33:39,447] {logging_mixin.py:115} INFO - [2023-12-03 21:33:39,447] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:33:39,447] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:33:39,455] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.023 seconds
[2023-12-03 21:34:12,068] {processor.py:153} INFO - Started process (PID=5973) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:34:12,069] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:34:12,069] {logging_mixin.py:115} INFO - [2023-12-03 21:34:12,069] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:34:12,080] {logging_mixin.py:115} INFO - [2023-12-03 21:34:12,080] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:34:12,080] {logging_mixin.py:115} INFO - [2023-12-03 21:34:12,080] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:34:12,080] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:34:12,090] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-03 21:34:43,819] {processor.py:153} INFO - Started process (PID=6008) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:34:43,820] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:34:43,821] {logging_mixin.py:115} INFO - [2023-12-03 21:34:43,820] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:34:43,831] {logging_mixin.py:115} INFO - [2023-12-03 21:34:43,831] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:34:43,831] {logging_mixin.py:115} INFO - [2023-12-03 21:34:43,831] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:34:43,831] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:34:43,841] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-03 21:35:15,765] {processor.py:153} INFO - Started process (PID=6050) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:35:15,765] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:35:15,765] {logging_mixin.py:115} INFO - [2023-12-03 21:35:15,765] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:35:15,776] {logging_mixin.py:115} INFO - [2023-12-03 21:35:15,776] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:35:15,776] {logging_mixin.py:115} INFO - [2023-12-03 21:35:15,776] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:35:15,777] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:35:15,786] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2023-12-03 21:35:50,594] {processor.py:153} INFO - Started process (PID=6088) to work on /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:35:50,595] {processor.py:641} INFO - Processing file /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-12-03 21:35:50,595] {logging_mixin.py:115} INFO - [2023-12-03 21:35:50,595] {dagbag.py:507} INFO - Filling up the DagBag from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:35:50,608] {logging_mixin.py:115} INFO - [2023-12-03 21:35:50,608] {example_local_kubernetes_executor.py:37} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 35, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-12-03 21:35:50,608] {logging_mixin.py:115} INFO - [2023-12-03 21:35:50,608] {example_local_kubernetes_executor.py:38} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-12-03 21:35:50,608] {processor.py:653} WARNING - No viable dags retrieved from /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-12-03 21:35:50,619] {processor.py:161} INFO - Processing /home/antosha/venvs/practice_3/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
